### PR TITLE
Update WinFormsFact version to include RetryCount for flaky tests

### PR DESCRIPTION
**Pull request by dreddy-work**
Test Infra:

Updating `WinFormsFact` version to include retry count on flaky UI tests.

### CODE REVIEW

File: src/Microsoft.VisualBasic.Forms/tests/UnitTests/Microsoft.VisualBasic.Forms.Tests.vbproj [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
+    </MSBuildWarningsAsMessages>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit.stafact" Version="$(XUnitStaFactPackageVersion)" />
+    <!-- xunit.stafact target TFM is contains "-windows" while our test binaris don't.
+    So, we had to use this work around to load correct binary from NuGet-->
+    <PackageReference Include="xunit.stafact" Version="$(XUnitStaFactPackageVersion)" ExcludeAssets="all"/>
+    <Reference Include="$(PkgXunit_StaFact)\lib\net6.0-windows7.0\xunit.stafact.dll" />
   </ItemGroup>
```
Comment (RussKie) [inline src/Microsoft.VisualBasic.Forms/tests/UnitTests/Microsoft.VisualBasic.Forms.Tests.vbproj:21]: These changes are repeated over and over, consider putting these in Directry.Build.props/.targets.

**Comment by dreddy-work** [comment]
Closing. Sam had different approach for work around and merged it.

**Pull request closed without merging**